src/database/models/product.js

```diff
@@ -0,0 +1,42 @@
+const Sequelize = require('sequelize');
+const db = require('./index').sequelize;
+
+const Products = db.define('products', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER,
+  },
+  productName: {
+    type: Sequelize.STRING,
+  },
+  productPrice: {
+    type: Sequelize.STRING,
+  },
+ productImage: {
+    type: Sequelize.STRING,
+  },
+  productDesc: {
+      type: Sequelize.STRING,
+  },
+  productCoupon: {
+    type: Sequelize.STRING,
+},
+productPromo: {
+    type: Sequelize.STRING,
+},
+  createdAt: {
+    allowNull: false,
+    type: Sequelize.DATE,
+  },
+  updatedAt: {
+    allowNull: false,
+    type: Sequelize.DATE,
+  },
+})
+freezeTableName = true;
```

Comment:
this isn't properly placed, should be placed inside an object just after the callback function of a model db.define('table_name',{},{
id:{
etc
}
freezeTableName:true})